src/api/verify.ts

```diff
@@ -49,27 +50,39 @@ class VerifyAPI {
       host: 'https://radar-verify.com:52516',
     });
 
-    const { user, events } = response;
+    const { user, events, token } = response;
     let location;
-    if (user.location && user.location.coordinates && user.locationAccuracy) {
+    if (user && user.location && user.location.coordinates && user.locationAccuracy) {
       location = {
         latitude: user.location.coordinates[1],
         longitude: user.location.coordinates[0],
         accuracy: user.locationAccuracy,
       };
     }
 
-    const trackRes = {
-      user,
-      events,
-      location,
-    } as RadarTrackResponse;
-
-    if (options.debug) {
-      trackRes.response = response;
+    if (encrypted) {
+      const trackTokenRes = {
+        token,
+      } as RadarTrackTokenResponse;
+  
+      if (options.debug) {
+        trackTokenRes.response = response;
+      }
+  
+      return trackTokenRes;
+    } else {
```

Comment:
Technically don't need this `else` since it returns above.

Reply:
Updated